compiler/test/runnable/mars1.d

```diff
@@ -2549,6 +2549,15 @@ void test9()
 
 ////////////////////////////////////////////////////////////////////////
 
+void test10()
+{
+    int v = 0x1234;
+    byte b = cast(ref byte)v;
+    assert(b == 0x34);
```

Comment:
this test assumes endianess, so will fail needlessly on some platforms

Reply:
fixed